frontends/mit-open/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -19,49 +18,147 @@ type SearchChip = {
 
 const SEARCH_CHIPS: SearchChip[] = [
   {
-    label: "AI Courses",
-    href: "/search?topic=Artificial+Intelligence&resource_type=course",
+    label: "New",
+    href: "/search?sortby=new",
```

Comment:
I know this isn't mentioned in the issue, but do we want to maybe filter this on "course" as well? For example, the "New" item in the nav menu points to `?resource_type=course&sortby=new` to make sure only courses are returned. Did we want to include all learning resources here or just courses?

Reply:
(or, maybe we should change the nav menu and remove the course filter, not sure... either way it should probably be consistent)

Reply:
We should change the nav drawer. Confirmed with Ferdi, and added to this PR.

https://github.com/mitodl/hq/issues/4179#issuecomment-2103190033